go/vt/tlstest/tlstest_test.go

```diff
@@ -80,20 +80,21 @@ func TestClientServer(t *testing.T) {
 	//
 	// Positive case: accept on server side, connect a client, send data.
 	//
-
+	var clientErr error
```

Comment:
Same concern about possible blocking in this test unless the Dial and Accept have timeouts.

Reply:
I fixed Dial, but can't use the same strategy for listener here because tls.listener is not exported.